{{ cookiecutter.project_name }}/test_requirements.txt

```diff
@@ -1,5 +1,4 @@
-astroid==2.0.4
-black==18.9b0
+black
```

Comment:
If we're adding the `black` dependency, should we also add a `pyproject.toml` file like xpro has? https://github.com/mitodl/mitxpro/blob/master/pyproject.toml

Reply:
Ah, good point